RecoTracker/TransientTrackingRecHit/python/TTRHBuilders_cff.py

```diff
@@ -7,7 +7,7 @@
 from RecoTracker.TkSeedingLayers.TTRHBuilderWithoutAngle4PixelPairs_cfi import *
 from RecoTracker.TkSeedingLayers.TTRHBuilderWithoutAngle4PixelTriplets_cfi import *
 #TransientTRH builder with template
-from RecoLocalTracker.SiPixelRecHits.pixelCPEFastESProducer_cfi import *
+from RecoLocalTracker.SiPixelRecHits.pixelCPEFastESProducerPhase1_cfi import *
```

Comment:
to spare me to do it in another PR changing just one line, would you mind adding the phase2 producer here as well?

see https://github.com/cms-sw/cmssw/pull/40215#discussion_r1039795040